source/hic_hal/nxp/lpc11u35/DAP_config.h

```diff
@@ -181,9 +187,21 @@ Disables the DAP Hardware I/O pins which configures:
 */
 static __inline void PORT_OFF(void)
 {
+#if !defined(PIN_nRESET_FET_DRIVE)
+    // open drain logic
     LPC_GPIO->DIR[PIN_nRESET_PORT] &= ~PIN_nRESET;
+#else
+    // FET drive logic
+    LPC_GPIO->DIR[PIN_nRESET_PORT] |= PIN_nRESET;
+    LPC_GPIO->CLR[PIN_nRESET_PORT] = PIN_nRESET;
+#endif
     LPC_GPIO->DIR[PIN_SWCLK_PORT] &= ~PIN_SWCLK;
     LPC_GPIO->DIR[PIN_SWDIO_PORT] &= ~PIN_SWDIO;
+
+#if defined(TARGET_POWER_HOLD)
+    //Release Target PowerHold
+    LPC_GPIO->CLR[PIN_PWH_PORT] = PIN_PWH;
+#endif
```

Comment:
I was confused about the purpose of PowerHold. I was thinking that clearing power hold turned off power to the MCU when really it just allows the MCU to turn off right? If so then your code is correct and you should ignore my first comment.